src/servers_non_y2/drbd.rb.yy

```diff
@@ -650,7 +656,7 @@ def writeFile()
     if $drbd.has_key?("global") then
       gccfile.puts "global {"
       $drbd["global"].each_key do |key|
-	    if key == "disable-ip-verification" then
+	    if key == "disable-ip-verification" or key == "udev-always-use-vnr" then
```

Comment:
I'm not sure how the .rb.yy files are processed, but in normal Ruby you do not need to use `then` (if the condition body is on a separate line).

Reply:
I think so, will verify before change.